src/content/docs/new-relic-solutions/observability-maturity/uptime-performance-reliability/error-optimization.mdx

```diff
@@ -0,0 +1,260 @@
+
+## Establish the current state [#establish-current-state]
+
+* [Create a workload for your applications](#create-workload)
+* [Create service levels for your workload](#create-service-level)
+
+### Create a workload for your applications [#create-workload]
+
+Define the list of applications and services for which you are trying to optimize errors. The team conducting the error optimization process should have full responsibility for and control of these apps and services. Once you have decided, set up a [workload](/docs/new-relic-solutions/new-relic-one/workloads/workloads-isolate-resolve-incidents-faster) for these entities.
+
+Workloads are a group of entities (applications, instances, etc.) for which a specific team is responsible. They allow you to look at data for only the entities you can do something about. We'll be basing most of our work going forward around the workload you set up here.
+
+It only takes a few minutes to set up a workload. See [the workload instructions](/docs/new-relic-solutions/new-relic-one/workloads/workloads-isolate-resolve-incidents-faster).  
+
+If you're already familiar with workloads and prefer to divide your applications and services into multiple workloads, you can. Just follow the steps below for each workload.
+
+### Create service levels for your workload [#create-service-level]
+
+[Service levels](/docs/service-level-management/intro-slm) allow you to easily configure and view [Service level objectives (SLOs)](https://sre.google/sre-book/service-level-objectives) for a given group of entities. Using service levels is one way you can monitor and communicate the success of your error management project.
+
+From your workload, navigate to the **Service levels** tab. Create a service level that measures error rates for each entity in the workload. This is configured in Step 2 in the service level UI. For each service level, use the WHERE clauses to filter out good requests or errors that shouldn't be factored in. 
+
+#### Create a service level for each APM service
+In step 2 of service creation, choose the **Success** SLI.
+
+#### Create a service level for each browser app
+In step 2 of service creation, choose the **Success** SLI.
+
+#### Create service levels for each mobile app
+* Create a service level for mobile crashes: In Step 2, choose `MobileSession` as the source of valid events. Choose `MobileCrash` as the source of bad responses.
+* Create a service level for mobile request errors: For Step 2, choose `MobileRequest` as the source of valid events. Choose `MobileRequestError` as the source of bad responses.
+
+### Create a service level for each serverless application
+Create an error rate service level for AWS Lambda function integrated with our [serverless monitoring](/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/monitoring-aws-lambda-serverless-monitoring):
```

Comment:
could you take another look at this sentence and make sure it's saying what you want?